documentation/usage/settings.md

```diff
@@ -2,8 +2,101 @@
 sidebar_position: 1
 ---
 
+import CaptionedImage from "/src/components/CaptionedImage";
+
 # Game Settings
 
+In addition to the settings available in the original Jak games, OpenGOAL provides some additional settings that will allow you to tweak and improve the games in new ways. This page will highlight some of these new settings and provide examples for them, but there are still more to find in game. Once you've had a look through this page, [try changing them yourself in game](/docs/usage/installation)!
+
+## Subtitles (and languages)
+
+**Found in Options > Game Options**
```

Comment:
just thinking out-loud, i imagine this will be a common thing in documentation -- might be nice to make a component that emphasizes it a bit better than just bold text

Reply:
Yeah I think that would make for a useful component for sure, a little breadcrumbs type display would be great. I might try and put something together the next time I add to the settings page, even if it's just a bit of styling